remoteappmanager/application.py

```diff
@@ -205,27 +205,13 @@ def _reverse_proxy_init(self):
 
     def _db_init(self):
         """Initializes the database connection."""
-        self.db = orm.Database(self.file_config.db_url)
+        self.db = orm.AccountingInterface(self.file_config.db_url)
```

Comment:
Don't worry about moving things around. We need to evolve a proper schema to organise the code.